test/libsolidity/smtCheckerTests/functions/functions_library_1.sol

```diff
@@ -17,5 +17,3 @@ contract C
 	}
 }
 // ----
-// Warning 8364: (228-229): Assertion checker does not yet implement type type(library L)
```

Comment:
Does this mean `L.add()` is silently ignored? If so, isn't it dangerous because it gives users the impression that assertion holds when in fact it may not but the SMTchecker does not model `L.add()`?

Reply:
No, just the `L` in `L.add()` is ignored. When handling the function call, we know which function it calls from the `FunctionCall` node, we don't need to resolve it here, it was already resolved before by something in `analysis`.

Reply:
@bshastry , you can see on the other tests, e.g. `functions/functions_library_1_fail.sol`, that the violation of the assertion is correctly detected.